spaceduck/src/Views/forms-style.css

```diff
@@ -8,12 +8,31 @@
  justify-content: center;
 
 }
+.button-container{
```

Comment:
wzoruj sie na from-container, to jest praktycznie ten sam widok, tylko inne buttony, mozesz uzyc tego samego stylu